sarif_cli/signature.py

```diff
@@ -56,6 +56,8 @@ def _signature_dict(args, elem, context: Context):
     if args.typedef_signatures:
         # Give every unique struct a name and use a reference to it as value.
         if signature not in context.sig_to_typedef:
+            #cannot have leading 0 hashes later in table joins so replace now
```

Comment:
is this relevant? if not perhaps rm

Reply:
that's just a whitespace change.  The comment is older, from

Reply:
I removed the comment.